src/lib/Specification/ContentType/ContentTypeIsDashboardContainer.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\AdminUi\Specification\ContentType;
+
+use Ibexa\AdminUi\Exception\InvalidArgumentException;
+use Ibexa\AdminUi\Specification\AbstractSpecification;
+use Ibexa\Contracts\Core\Repository\Values\ContentType\ContentType;
+
+class ContentTypeIsDashboardContainer extends AbstractSpecification
```

Comment:
ibexa/admin-ui must not have any knowledge about the ibexa/dashboard

Reply:
This is not resolved, you are still checking for `dashboard.container_content_type_identifier` parameter. What you need here is an extension point and contain that whole logic inside dashboard package.